bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1265,9 +1267,10 @@ private void checkStatus() {
             } else {
                 // Determine current status
                 jcheckType = "jcheck";
-                var additionalConfiguration = AdditionalConfiguration.get(localRepo, localHash,
+                var conf = checkablePullRequest.parseJCheckConfiguration(localRepo, jcheckConfHash);
+                var additionalConfiguration = AdditionalConfiguration.get(conf,
                         pr.repository().forge().currentUser(), comments, reviewMerge);
-                checkablePullRequest.executeChecks(localHash, censusInstance, visitor, additionalConfiguration, checkablePullRequest.targetHash());
+                checkablePullRequest.executeChecks(localHash, censusInstance, visitor, additionalConfiguration, jcheckConfHash);
                 // Don't need to run the second round if confOverride is set.
```

Comment:
Strictly speaking, should we regenerate the AdditionalConfiguration for the second run where we use the merged .jcheck/conf?

Reply:
Oh, right, I overlooked it when I was implementing the second run of jcheck feature.

Reply:
This has been re-worked in the latest version